x-pack/plugins/cases/server/services/index.ts

```diff
@@ -5,1151 +5,15 @@
  * 2.0.
  */
 
-import { cloneDeep } from 'lodash';
-import {
-  KibanaRequest,
-  Logger,
-  SavedObject,
-  SavedObjectsClientContract,
-  SavedObjectsFindResponse,
-  SavedObjectsUpdateResponse,
-  SavedObjectReference,
-  SavedObjectsBulkUpdateResponse,
-  SavedObjectsBulkResponse,
-  SavedObjectsFindResult,
-} from 'kibana/server';
+import { SavedObjectsClientContract } from 'kibana/server';
```

Comment:
Is this import needed?

Reply:
Is needed for

```
export interface ClientArgs {
  soClient: SavedObjectsClientContract;
}
```

that lies in the same file. I am missing something?